test/trim_trailing_whitespaces_test.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require_relative 'lib'
+
+describe 'Trim trailing whitespaces tests' do
+  it 'does not trim non trailing whitespaces' do
+    printer = Oppen::Wadler.new(width: 5)
+    printer.text('a ')
+    printer.text('b')
+    _(printer.output).must_equal 'a b'
```

Comment:
Add a whitespace at before `a` and before `b`.

Reply:
Change applied to the second commit